src/nextcloud/talk/recording/Service.py

```diff
@@ -176,6 +179,10 @@ def start(self, actorType, actorId):
 
         fullDirectory = f'{directory}/{sanitizedBackend}/{self.token}'
 
+        # For linux/aarch64 aka arm64, the automatic finding mechanism of the executable will fail (unsupported by selenium in the specified version)
+        # Solution: make it configurable.
+        executable_path = config.getBackendDriver(self.backend)
```

Comment:
As it is directly tied to the browser being used, I think it would be better if getting the driver was done [right after getting the browser](https://github.com/nextcloud/nextcloud-talk-recording/blob/10e43852d812df41dbba93e865e79aa2020c0186/src/nextcloud/talk/recording/Service.py#L206).